frontend/javascripts/oxalis/view/settings/user_settings_view.js

```diff
@@ -445,6 +445,11 @@ class UserSettingsView extends PureComponent<UserSettingsViewProps> {
             value={this.props.userConfiguration.dynamicSpaceDirection}
             onChange={this.onChangeUser.dynamicSpaceDirection}
           />
+          <SwitchSetting
+            label={settingsLabels.useLegacyBindings}
```

Comment:
What do you think about a small description of what this entails? Or do we simply say, the "old" people know what this means (and have it activated automatically) and nobody else should activate it?

Reply:
Thinking about this: Existing organizations may want it that this setting is automatically set even for newly registered users, because they have training material referencing the legacy mouse/keyboard handlers.

Reply:
> What do you think about a small description of what this entails? Or do we simply say, the "old" people know what this means (and have it activated automatically) and nobody else should activate it?

Good idea! I think, this definitely needs an explanation, so the user isn't left puzzled. I'll add one.

> Thinking about this: Existing organizations may want it that this setting is automatically set even for newly registered users, because they have training material referencing the legacy mouse/keyboard handlers.

Hmm, good point. If we are lucky we could count on the recommended user config which should also cover new users. However, a proper organization-wide setting would probably be safer.. I'll add this to the todo list.